src/scenarios/init.ps1

```diff
@@ -27,7 +27,7 @@ function Setup-Env($directory){
 function Download-Dotnet($channel){
     Write-Host "Downloading dotnet from channel $channel"
     $dotnetScript= Join-Path "$scripts" 'dotnet.py' -Resolve
-    python $dotnetScript install --channels $channel -v
+    py -3 $dotnetScript install --channels $channel -v
```

Comment:
Why is this necessary?

Reply:
The original command wasn't running properly on my machine, and Drew suggested doing this general change to make things work reliably.